### PR TITLE
Update faq.rst

### PR DESCRIPTION
**Pull request by SagevdBrand**
updating based on questions in workshop.

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -220,6 +222,14 @@ can indicate previous screening decisions on records with 0s
 (irrelevant) and 1s (relevant). ASReview will use this information to
 train the model.
 
+**Related question: How can I add more publications while I are already started
```
Comment (J535D165) [inline docs/source/faq.rst:225]: Please use the underline and put everything on a single line.

File: docs/source/faq.rst [hunk 2]
```diff
@@ -211,7 +213,7 @@ different classifiers on several datasets
 We do not advise specific classifiers for specific jobs because we've not
 found enough evidence (yet) to make such recommendations.
 
-I had already started labeling before I came across ASReview. How can I keep my former screening decisions when starting a new project in ASReview?
+I already started labeling before I came across ASReview. How can I keep my former screening decisions when starting a new project in ASReview?
```
Comment (J535D165) [inline docs/source/faq.rst:216]: Please check whether the underline has the same length.

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks! 2 remarks.

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks for the changes. I have to be strict. Otherwise the docs won't render well.

**Comment by GerbrichFerdinands** [comment]
Awesome, thanks @SagevdBrand!

**Pull request merged**